### PR TITLE
add coverage to workflows

### PR DESCRIPTION
**Pull request by MP91**
## Describe your changes

adds a unit test coverage report to the workflow

## Checklist - Manual tasks

* [ ] Examples are executing successfully
* [ ] Created/updated unit tests. Code Coverage percentage on new code shall be >= 80%.
* [ ] Created/updated integration tests.
* [ ] Devcontainer can be opened successfully
* [ ] Devcontainer can be opened successfully behind a corporate proxy
* [ ] Devcontainer can be re-built successfully
* [ ] Extended the documentation (e.g. README.md, CONTRIBUTING.md, Velocitas)

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -15,8 +15,10 @@ python =
 [testenv]
 deps = -rrequirements.txt
 commands =
-    pytest
-
+    pytest --override-ini junit_family=xunit1 --junit-xml=./results/UnitTest/junit.xml \
+          --cov ./sdv \
```
Comment (kse3hi) [inline tox.ini:19]: Shall we exclude sdv/proto out of coverage report?

Comment (MP91) [reply tox.ini:19]: hmm not sure, I mean technically we did not write the files and therefore don't need to test them, but on the other hand they are the base for everything...

Comment (MP91) [reply tox.ini:19]: @mml5bg @BjoernAtBosch @doosuu  what do you think about the generated proto files?

Comment (kse3hi) [reply tox.ini:19]: I am ok to have integration test to verify existence of generated files, probably version or some particular content of them, but do we need include line and branch coverage of generated files - sounds strange :)

Comment (BjoernAtBosch) [reply tox.ini:19]: For me it also feels strange to test generated source code files. This should be covered by testing the generator - in our case the protoc (grpc), where I would expect it to be well-tested already.

Comment (mml5bg) [reply tox.ini:19]: I agree that is strange to test generated source code files, but existence of them or particular version would be good and enough.

Comment (MP91) [reply tox.ini:19]: excluded the proto-directory from the coverage calculation and report

**Review by mml5bg — APPROVED** [review]
LGTM

**Pull request merged**